docs/kubernetes/deploy.md

```diff
@@ -52,13 +52,14 @@ INFO[0393] Finished ARM Deployment (contoso-apple-1423145182).
 
 `acs-engine` will output Azure Resource Manager (ARM) templates, SSH keys, and a kubeconfig file in `_output/contoso-apple-59769a59` directory:
 
-   * `_output/contoso-apple-59769a59/azureuser_rsa`
-   * `_output/contoso-apple-59769a59/kubeconfig/kubeconfig.westus2.json`
+* `_output/contoso-apple-59769a59/azureuser_rsa`
+* `_output/contoso-apple-59769a59/kubeconfig/kubeconfig.westus2.json`
 
 acs-engine generates kubeconfig files for each possible region. Access the new cluster by using the kubeconfig generated for the cluster's location. This example used `westus2`, so the kubeconfig is `_output/<clustername>/kubeconfig/kubeconfig.westus2.json`:
 
-```
-$ KUBECONFIG=_output/contoso-apple-59769a59/kubeconfig/kubeconfig.westus2.json kubectl cluster-info
+```sh
+$ KUBECONFIG=_output/contoso-apple-59769a59/kubeconfig/kubeconfig.westus2.json
+$ kubectl cluster-info
```

Comment:
These two commands should stay as a one-liner, or the first command should begin with `export KUBECONFIG=` so the environment variable is visible to the second command.